src/handle.ml

```diff
@@ -170,7 +172,7 @@ let handle_cmd_aux : sig_state -> command -> sig_state * proof_data option =
             if not (Proof.finished st) then
               fatal cmd.pos "The proof is not finished.";
             (* Add a symbol corresponding to the proof. *)
-            let s = Sign.add_symbol ss.signature Const x a in
+            let s = Sign.add_symbol ss.signature Const x a [] in
```

Comment:
Idem here: Implicits are missing here. There may be implicit arguments in theorems too.

Reply:
Done, thanks.

Reply:
Idem: Where are the implicits? It shouldn't be [].